libopenage/renderer/README.md

```diff
@@ -0,0 +1,149 @@
+# Openage graphics
+The graphics subsystem is implemented in two levels. The first level is an abstraction over graphics APIs (OpenGL, Vulkan) and provides generic shader execution methods. The second level uses the first to draw openage-specific graphics, i.e. the actual world, units, etc.
+
+### Namespaces:
+`openage::renderer` - the level 1 renderer
+`openage::renderer::opengl` - the OpenGL implementation
+`openage::renderer::vulkan` - the Vulkan implementation
+`openage::renderer::resources` - management of graphics assets
+
+__TODO name__
+`openage::graphics` - the level 2 system
+
```

Comment:
It's uncertain yet whether there would be level 2 and 3 or whether we can just have the game rendering code and general presentation code in one unit, which would just be level 2. I think it depends on how much code there will be, so that if it's little we can have one unit, but if there's a lot two levels would be better.